Resources/Private/Fusion/Root.fusion

```diff
@@ -1,8 +1,31 @@
 // this is only included in dev context
+// todo not anymore
 prototype(Neos.Fusion:Component) {
     @exceptionHandler = 'PackageFactory\\AtomicFusion\\PropTypes\\Error\\ExceptionHandler\\PropTypeExceptionHandler'
 }
 
 prototype(PackageFactory.AtomicFusion:Component) {
     @exceptionHandler = 'PackageFactory\\AtomicFusion\\PropTypes\\Error\\ExceptionHandler\\PropTypeExceptionHandler'
 }
+
+prototype(PackageFactory.AtomicFusion.PropTypes:PropsFromComponent) < prototype(Neos.Fusion:Value) {
```

Comment:
naming is WIP too - when we find a fitting name i will extract the prototypes into its own folders

Reply:
i find `PackageFactory.AtomicFusion.PropTypes:ForComponentCollection` tempting, as i would name my php collection dto like that...

or we could also play with the words `infer` and `props` - they are common in typescript / js

or similar to `React.ComponentProps<MyComponent>`?

Reply:
`Prop.Types:FromComponent`
`PackageFactory.AtomicFusion.PropTypes:PropTypesFromComponent`

we could also use a prop `isIterateable = true` instead of adding a second prototype for array